lib/Core/readTemplate.js

```diff
@@ -0,0 +1,45 @@
+"use strict";
+/**
+ * Parse inforTemplate
+ * @param  {entity.description.getRawValue()} data raw data of of the picked feature
+ * @param  {inforTemplate} template info template as defined in the info file
+ * @return {String}  Html to be displayed in the info box
+ */
+var readTemplate = function (data, template) {
+    var html = "",
+        rg = /{{2}([^}]+)}{2}/g;
+
+    function keyFilter(string) {
+        //use {{}} syntax
+        return function handler(regex) {
+            return string.match(regex);
+        };
+    }
+    //if supplied a string
+    function parse(template) {
+        var keys,
+            output ='',
+            currentKey,
+            currentValue,
+            i = 0;
+        keys = (keyFilter(template))(rg);
```

Comment:
I'm confused about the purpose of this function call.  Isn't this line equivalent to `keys = template.match(rg)`?